recipes/koopa-viz/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "koopa-viz" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/koopa-viz-{{ version }}.tar.gz
+  sha256: f7bf948c3a08d8f854ba42c48c7a0b49d5f7e6fde92db93dce9f87166b9312e0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools
+  run:
+    - napari
+    - numpy
+    - pandas
+    - pyarrow
+    - python >=3.8
+    - qtpy
+    - scikit-image
+    - tifffile
+    - trackpy
+
+test:
+  imports:
+    - koopa_viz
+  commands:
+    - pip list
```

Comment:
Why is this needed? Otherwise lgtm.
```suggestion
```

Reply:
Hi @xhochy it is just a quality check that helps debug the package creation, for example when setuptools-scm is omitted pip list will reveal packages with version number 0.0.0, but still succeed. 

I have been using this in a lot of recipes and has made life easier :)